packages/eslint-plugin/src/util/collectUnusedVariables.ts

```diff
@@ -439,6 +439,17 @@ function isExported(variable: TSESLint.Scope.Variable): boolean {
   });
 }
 
+const LOGICAL_ASSIGNMENT_OPERATORS = new Set(['&&=', '||=', '??=']);
+
+/**
+ * Checks whether the given operator is a logical assignment operator or not.
+ * @param operator The operator
```

Comment:
[Proofreading] Just nitpicking a bit: `@param operator The operator` gives no new information and just takes up space.

* I wrote a lint rule for this 😄 https://github.com/JoshuaKGoldberg/are-docs-informative ->  https://github.com/gajus/eslint-plugin-jsdoc/blob/aefceca7413ac0263ed1b58127fff4efca89de4f/docs/rules/informative-docs.md#readme.
* Filed #7855 to lint for it automatically

```suggestion
```